src/type_checker/resources/std_lib/range.py

```diff
@@ -0,0 +1,10 @@
+class range:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'range'](https://app.codacy.com/app/JSAbrahams/mamba/pullRequest?prid=4098440)